python/private/common_labels.bzl

```diff
@@ -0,0 +1,27 @@
+"""Constants for common labels used in the codebase."""
+
+# NOTE: str() is called because some APIs don't accept Label objects
+# (e.g. transition inputs/outputs or the transition settings return dict)
+
+labels = struct(
+    # keep sorted
+    ADD_SRCS_TO_RUNFILES = str(Label("//python/config_settings:add_srcs_to_runfiles")),
+    BOOTSTRAP_IMPL = str(Label("//python/config_settings:bootstrap_impl")),
+    EXEC_TOOLS_TOOLCHAIN = str(Label("//python/config_settings:exec_tools_toolchain")),
```

Comment:
Having
```
_labels = dict(
    ...
)

labels = struct(
    all = _labels.values(),
    **_labels,
)
```

would be nicer, I think.

Reply:
I was intending for this to have more than just flags (e.g. `:current_py_cc_headers` et al) basically any label we end up typing more than 3 times, so I don't think `all()` makes sense in that context.

Hm, REPL_DEP...I don't think it makes sense to have py_binary etc transition on that one. The repl targets are just intended for interactive usage. I've removed that from transition_labels.